$OpenBSD: patch-src_context_c,v 1.2 2015/11/13 02:48:29 gsoares Exp $

merged upstream
github.com/brunoos/luasec/commit/49ea6b8ba654f118fbb158da098e6ef288519126

--- src/context.c.orig	Sat Jul 18 11:36:23 2015
+++ src/context.c	Sat Jul 18 11:42:12 2015
@@ -66,7 +66,9 @@ static int set_option_flag(const char *opt, unsigned l
 static LSEC_SSL_METHOD* str2method(const char *method)
 {
   if (!strcmp(method, "sslv23"))  return SSLv23_method();
+#ifndef OPENSSL_NO_SSL3
   if (!strcmp(method, "sslv3"))   return SSLv3_method();
+#endif
   if (!strcmp(method, "tlsv1"))   return TLSv1_method();
 #if (OPENSSL_VERSION_NUMBER >= 0x1000100fL)
   if (!strcmp(method, "tlsv1_1")) return TLSv1_1_method();
